app/models/controlled_work_document_selection.rb

```diff
@@ -16,7 +16,16 @@ class ControlledWorkDocumentSelection
                        if: -> { FeatureFlags.enabled?(:welsh_cw, without_session_data: true) }
 
   def options
-    OPTIONS.select { !check.asylum_support || %i[cw1 cw2].include?(_1) }
-           .map { { value: _1, options: { label: { text: I18n.t("controlled_work_document_selections.new.option.#{_1}") } } } }
+    if client_under_18_feature_flag_and_clr
+      OPTIONS.select { %i[cw2 cw1_and_2].include?(_1) }
```

Comment:
There's a lot of repetition in the two halves of this if/else. Ideally we'd do something like:

```ruby
relevant_forms = if check.controlled_legal_representation
                   %i[cw2 cw1_and_2]
                 elsif check.asylum_support
                   %[cw1 cw2]
                 else
                   OPTIONS
                 end

relevant_forms.map { { value:...
```

That way all the `map`ping and `I18n.t`ing only happens once.